src/main/resources/db/changelog/v.0.6/db.changelog-add-discounts-image-data.yaml

```diff
@@ -0,0 +1,197 @@
+databaseChangeLog:
+  - changeSet:
+      id: 0.6-add-discounts-image-data
+      author: artem_kishenets
+      changes:
+        - insert:
+            tableName: discounts_image
+            columns:
+              - column:
+                  name: id
+                  value: "06238eac-e612-46fb-9f9a-6705c42948e4"
+              - column:
+                  name: discount_id
+                  value: "5f69268b-705e-4fb9-8147-722b4ec1d9da"
+              - column:
+                  name: image
+                  value: "https://exdadel-discounts-image-upload.s3.eu-central-1.amazonaws.com/discounts-images/yoga.jpg"
+              - column:
+                  name: by_default
+                  value: true
+        - insert:
+            tableName: discounts_image
+            columns:
+              - column:
+                  name: id
+                  value: "06238eac-e612-46fb-9f9a-6705c42948e5"
+              - column:
+                  name: discount_id
+                  value: "5f69268b-705e-4fb9-8147-722b4ec1d9da"
+              - column:
+                  name: image
+                  value: "https://exdadel-discounts-image-upload.s3.eu-central-1.amazonaws.com/discounts-images/yoga1.jpg"
```

Comment:
add 
- column:
                  name: by_default
                  value: true/false
                  
please

Reply:
No 😿

Reply:
Actually yes, thanks for this 😼